cli/src/main/java/dev/starfix/starfix/Config.java

```diff
@@ -1,4 +1,4 @@
-package dk.xam.starfish;
+package dev.starfix.starfix;
```

Comment:
sorry for not noticing this first time around but shouldn't this just be `dev.starfix` and not `dev.starfix.starfix` ?